microcosm_logging/factories.py

```diff
@@ -113,20 +112,17 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
+    fields = """{
         "loggerName": "%(name)s",
         "levelName": "%(levelname)s",
-        "message": "%(message)s",
```

Comment:
How do you specify a custom Formatter using `logging.config.dictConfig`? I see you can specific a custom class for handlers but not for formatters.